### ISSUE TITLE
can't sync two vods at the same time 

### ISSUE THREAD

**Issue by Poli93** [henryperson/twitchmultivod#27]
https://twitchmultivod.com/#/1705639530/1705381956?t=6h15m25s

When trying to sync both of these vods at the same time it always has a 1 second delay, is this intended? 

(see the countdown isn't synced) 
![image](https://user-images.githubusercontent.com/965110/213506179-e99c92df-ebc2-450a-9755-b1bd6672327b.png)

**Comment by henryperson** [comment]
Yeah, unfortunately because of issues with twitch's API I sometimes cannot get granular enough to get an exact sync. There seems to be a bit of a margin for error in when twitch says a video starts (which is how I sync them—look at when they started in real time to find the real time of when you are trying to sync to, then move all other videos to that real time). So for some combinations of video you can only get a second or two away, sadly
